### PR TITLE
Export of multiple sequence diagrams

### LINKED ISSUE #583: PlantUML export doesn't work if a CompilationUnit contains more than one sequence diagrams
If a CompilationUnit contains more than one sequence diagrams, only the diagram with the same name as the CompilationUnit is visualized and it is not exported properly.

### PR DESCRIPTION
**Pull request by jtamas96**
This pull request fixes #583.

Do not merge this before #564  and #574.

### CODE REVIEW

**Comment by kmozsi** [comment]
Jenkins, retest this, please.

File: dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/wizards/PlantUMLVisualizeWizard.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+							Dialogs.errorMsgb("txtUML export Error", "Error occured during the PlantUml exportation.",
+									e);
+							monitor.done();
+							throw new InterruptedException();
+						}
+					}
+
+				}, ResourcesPlugin.getWorkspace().getRoot());
+			} catch (InterruptedException | InvocationTargetException e) {
+				e.printStackTrace();
```
Comment (kmozsi) [inline dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/wizards/PlantUMLVisualizeWizard.java]: Is it needed or just left here after debugging?

**Comment by kmozsi** [comment]
If the export of multiple diagrams is supported, the execution of the same diagram descriptions should be also possible. I have opened #600 for this issue.

File: dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/wizards/PlantUMLVisualizeWizard.java (hunk 1 above)
Comment (jtamas96) [reply dev/plugins/hu.elte.txtuml.export.plantuml/src/hu/elte/txtuml/export/plantuml/wizards/PlantUMLVisualizeWizard.java]: I just forgot to delete it after debugging.

**Pull request merged**